keps/sig-testing/2420-reducing-kubernetes-build-maintenance/README.md

```diff
@@ -125,9 +125,9 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [x] (R) Design details are appropriately documented
 - [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [x] (R) Graduation criteria is in place
-- [ ] (R) Production readiness review completed
+- [x] (R) Production readiness review completed
```

Comment:
/cc @johnbelamaric @wojtek-t @deads2k 

We believe PRR is not relevant to this KEP, given that it's build-time tech debt removal, and this will result in no change to how kubernetes release artifacts are produced/consumed.

Reply:
/hold
for ack

Reply:
er also I may have misread this particular checkbox: I read this as "filled out the template" and the checkbox below as "the contents of filling out the template were approved" (the checkbox below is not checked currently).